docs/user-guide/merge-translations.md

```diff
@@ -0,0 +1,59 @@
+# Merge translations from another project version
+
+Merge translations allows project maintainers to copy translations across different project version based on matching context. (See below for context matching rules)
+
+## Restrictions
+
+- This feature only available to project maintainers.
+- Only translations that are in translated/approved state will be used.
+- Merge translation can only be run if there are no other copy operations in progress for the selected version, such as copy-trans or copy version.
+
+## Rule of which translations will be copied over
+<table class='docutils'>
+    <tr>
+        <td>**From**</td><td>**To**</td><td>**Copy?**</td>
+    </tr>
+    <tr>
+        <td>fuzzy/untranslated</td><td>any</td><td>No</td>
+    </tr>
+    <tr>
+        <td>different source text/document id/<br />resId/msgctxt/locale</td><td>any</td><td>No</td>
+    </tr>
+    <tr>
+        <td>translated/approved</td><td>untranslated/fuzzy</td><td>Yes</td>
+    </tr>
+     <tr>
+        <td>translated/approved</td><td>translated/approved</td><td>copy if `From` is newer and <br />`Keep existing translated/approved translations` is unchecked</td>
+    </tr>
+</table>
+     
+## Run Merge translations
+
+1. Login to Zanata.
+1. Select a project version of which you wish to copy translations to.
```

Comment:
"Select a project version you wish to copy translations to." (current line does not make grammatical sense)